src/boardlib/api/aurora.py

```diff
@@ -21,6 +21,8 @@
 }
 
 GRADES = {
+    10: "4A",
+    11: "4B",
```

Comment:
I had done some 0s too but I guess they were 4C 😄 . Looks like this change should cover everything, just checked the app and 4A is the minimum grade